src/static/css/content.less

```diff
@@ -279,6 +276,10 @@ figure {
 
```

Comment:
The screenshot is caused by the tint table. If you delete it from the page, this section goes back to what it should be. The table itself forces the body wider than it should be.